### ISSUE TITLE
Another auto-versioning issue with Maven

### ISSUE THREAD

**Issue by jrudolph** [akka/akka-grpc#332]
Travis test on current master fails:

https://travis-ci.org/akka/akka-grpc/jobs/413520733:

```
[WARNING] The POM for com.lightbend.akka.grpc:akka-grpc-maven-plugin:jar:0.2+-fff5651b is missing, no dependency information available
```

while the actual version was `0.2-RC1`.

So, this was actually the tagged version, however, commits after that fail as well:

https://travis-ci.org/akka/akka-grpc/jobs/413534300

```
[WARNING] The POM for com.lightbend.akka.grpc:akka-grpc-maven-plugin:jar:0.2+-a693ac3d is missing, no dependency information available
```

for version `0.2-RC1+5-a693ac3d`.

**Comment by jrudolph** [comment]
For the same second version, Gradle version is broken again as well:

https://travis-ci.org/akka/akka-grpc/jobs/413534302

```
Could not determine the dependencies of task ':compileJava'.
> Could not resolve all task dependencies for configuration ':compileClasspath'.
   > Could not find com.lightbend.akka.grpc:akka-grpc-runtime_2.12:0.2+RC1+5-a693ac3d.
     Searched in the following locations:
         file:/home/travis/.m2/repository/com/lightbend/akka/grpc/akka-grpc-runtime_2.12/0.2+RC1+5-a693ac3d/akka-grpc-runtime_2.12-0.2+RC1+5-a693ac3d.pom
         file:/home/travis/.m2/repository/com/lightbend/akka/grpc/akka-grpc-runtime_2.12/0.2+RC1+5-a693ac3d/akka-grpc-runtime_2.12-0.2+RC1+5-a693ac3d.jar
         https://repo.maven.apache.org/maven2/com/lightbend/akka/grpc/akka-grpc-runtime_2.12/0.2+RC1+5-a693ac3d/akka-grpc-runtime_2.12-0.2+RC1+5-a693ac3d.pom
         https://repo.maven.apache.org/maven2/com/lightbend/akka/grpc/akka-grpc-runtime_2.12/0.2+RC1+5-a693ac3d/akka-grpc-runtime_2.12-0.2+RC1+5-a693ac3d.jar
     Required by:
         project :
```

So, it's looking at `0.2+RC1+5-a693ac3d`

**Comment by ktoso** [comment]
Oh man this version gen is always so confusing.... :-( I'd love the boring `version.sbt` style...

**Comment by jrudolph** [comment]
Didn't get anywhere with this.

Here's a script which returns the version number from sbt:

```
sbt 'set TaskKey[Unit]("showVersion") := println(s"Version #${version.value}#")' showVersion | grep "Version #" | sed -r -e 's/Version #([^#]*)#/\1/'
```

I guess what we could do is add a task to sbt which just puts the version number down on disk to be picked up by the maven and gradle plugins?

**Comment by jrudolph** [comment]
Fixed in #334 (for now ;) ).
